pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -452,3 +464,22 @@ func (v *vmValidator) checkReservedMemoryAnnotation(vm *kubevirtv1.VirtualMachin
 func (v *vmValidator) checkStorageResourceQuota(vm *kubevirtv1.VirtualMachine, oldVM *kubevirtv1.VirtualMachine) error {
 	return v.rqCalculator.CheckStorageResourceQuota(vm, oldVM)
 }
+
+func (v *vmValidator) checkDedicatedCPUPlacement(vm *kubevirtv1.VirtualMachine) error {
+	if !isDedicatedCPU(vm) {
+		return nil
+	}
+
+	selector := labels.Set{kubevirtv1.CPUManager: "true"}.AsSelector()
+	nodeList, err := v.nodeCache.List(selector)
+	if err != nil {
+		return fmt.Errorf("failed to list nodes with labels %s: %v", selector.String(), err)
+	}
+
+	if len(nodeList) == 0 {
+		return werror.NewInvalidError("no nodes with enabled CPU Manager found",
```

Comment:
no nodes with enabled CPU Manager found
>>
VM requets CPU pinning, but no nodes with enabled CPU Manager are/were found
..., did not find any node with enabled CPU Manager